kong/plugins/aws-lambda/handler.lua

```diff
@@ -0,0 +1,176 @@
+local cjson = require 'cjson'
+local http_client = require 'kong.tools.http_client'
```

Comment:
Ran into this while quickly glancing over this code. This http_client should definitely NOT be used, it uses LuaSocket under the hood, and hence uses blocking socket connections.

That client is to be used for testing only (and was actually already removed in 0.9)